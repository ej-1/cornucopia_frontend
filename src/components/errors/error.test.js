// Link.react.test.js
import React from "react";
import Error from "../errors/error";
import renderer from "react-test-renderer";

test("Error message gets rendered", () => {
  const component = renderer.create(<Error message={"Danger! Danger!"} />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
