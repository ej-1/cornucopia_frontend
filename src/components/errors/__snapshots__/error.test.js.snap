// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error message gets rendered 1`] = `
<div
  className="error"
>
  Danger! Danger!
</div>
`;
